components/cache.rst

```diff
@@ -23,112 +24,119 @@ Installation
 
 .. include:: /components/require_autoload.rst.inc
 
-Cache (PSR-6) Versus Simple Cache (PSR-16)
-------------------------------------------
+Cache Contracts versus PSR-6
```

Comment:
In the docs, we also try to be as backwards compatible as possible. In this case, that means that whenever we change a title, we should make sure the old title anchor still works by adding it manually above the title:

```rst
.. _cache-psr-6-versus-simple-cache-psr-16:

Cache Contracts versus PSR-6
----------------------------
```

Reply:
In the anchor PSR 16 is written instead of PSR 6